templates/navigation/oe-corporate-blocks-ec-footer--standardised.html.twig

```diff
@@ -11,109 +11,94 @@
  */
 #}
 
-{% set _section_2 = [] %}
-{% set _section_3 = [] %}
+{% set _top_left_column = [] %}
+{% set _top_right_column = [] %}
 
 {# First we parse the other_links, we make a simple grid. #}
 {% for section in site_specific_footer.other_links %}
   {% if loop.index is odd %}
-    {% set _section_2 = _section_2|merge([{
+    {% set _top_left_column = _top_left_column|merge([{
       'title': section.label,
+      'title_with_separator': true,
       'links': ecl_footer_links(section.links),
-      'section_id': 2,
-      'title_class_name': 'ecl-footer-standardised__title--separator',
     }]) %}
   {% else %}
-    {% set _section_3 = _section_3|merge([{
+    {% set _top_right_column = _top_right_column|merge([{
       'title': section.label,
+      'title_with_separator': true,
       'links': ecl_footer_links(section.links),
-      'section_id': 3,
-      'title_class_name': 'ecl-footer-standardised__title--separator',
     }]) %}
   {% endif %}
 {% endfor %}
 
 {# When follow section is present. #}
 {% if site_specific_footer.social_links is not empty %}
   {# If section numbers are odd, then we flip the last row so to have the follow as first. #}
-  {% if _section_2|length > _section_3|length %}
-    {% set _last_in_section_2 = _section_2|last %}
+  {% if _top_left_column|length > _top_right_column|length %}
```

Comment:
The social media icons on the standardised EC footer seem to be full white while [here](https://v3--europa-component-library.netlify.app/playground/ec/?path=/story/components-footers-standardised--default) they look different. Do we need to update to a newer version?
<img width="404" alt="social icons ec standardised" src="https://user-images.githubusercontent.com/22908988/122017398-99a8da80-cdca-11eb-8b67-ffc1fe65dbbe.png">

Reply:
You are right, they are different, we should be using the "negative" variant.